addons/sale/sale.py

```diff
@@ -616,7 +615,7 @@ def _action_procurement_create(self):
 
     @api.model
     def _get_analytic_invoice_policy(self):
-        return ['cost']
+        return ['delivery']
```

Comment:
This is a bit weird because if you create a SO with a line whose product is 'invoice on delivery', you won't get an analytic account; but if you add a line to a *confirmed* so with the same product, you **will** get an analytic account.

I'll probably remove `_get_analytic_invoice_policy' and `_get_analytic_track_service` and create the analytic account explicitely when needed (I wanted to factorize even more, but since after talking with antony I'd rather remove it completely)